<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>БД+</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../media/add_data_to_base_rs.qrc">
    <normaloff>:/icon/add_data.ico</normaloff>:/icon/add_data.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>561</width>
        <height>575</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="previousButton">
           <property name="text">
            <string>Пред.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextButton">
           <property name="text">
            <string>След.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Сохр.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="cursor">
          <cursorShape>SizeVerCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="title">
          <string>Основные данные</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="labelAlignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Артикул:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="articul_lineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Модель:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Есть ли ограничитель:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="check_limiter_comboBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
            <item>
             <property name="text">
              <string>Нет</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Да</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Ограничитель:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="limiter_comboBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="add_name_store_pushButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>10</horstretch>
                <verstretch>10</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>10</width>
                <height>10</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../media/add_data_to_base_rs.qrc">
                <normaloff>:/icon/plus.ico</normaloff>:/icon/plus.ico</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="remove_name_store_pushButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../media/add_data_to_base_rs.qrc">
                <normaloff>:/icon/minus.ico</normaloff>:/icon/minus.ico</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <widget class="QWidget" name="sity_1">
          <attribute name="icon">
           <iconset resource="../media/add_data_to_base_rs.qrc">
            <normaloff>:/icon/city.ico</normaloff>:/icon/city.ico</iconset>
          </attribute>
          <attribute name="title">
           <string>Город 1</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Текущая цена:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="current_price_lineEdit_1">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="first_cost__label">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Себестоимость:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="first_cost_lineEdit_1"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Минимальная цена:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="min_price_lineEdit_1">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>%</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="min_price_spinBox_1">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
               <property name="value">
                <double>25.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_10">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Максимальная цена:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLineEdit" name="max_price_lineEdit_1">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>%</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="max_price_spinBox_1">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
               <property name="value">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="sity_2">
          <attribute name="icon">
           <iconset resource="../media/add_data_to_base_rs.qrc">
            <normaloff>:/icon/city.ico</normaloff>:/icon/city.ico</iconset>
          </attribute>
          <attribute name="title">
           <string>Город 2</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_3">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Текущая цена:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="current_price_lineEdit_2">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Себестоимость:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="first_cost_lineEdit_2"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Минимальная цена:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLineEdit" name="min_price_lineEdit_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>%</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="min_price_spinBox_2">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
               <property name="value">
                <double>25.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Максимальная цена:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="max_price_lineEdit_2">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>%</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="max_price_spinBox_2">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
               <property name="value">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="sity_3">
          <attribute name="icon">
           <iconset resource="../media/add_data_to_base_rs.qrc">
            <normaloff>:/icon/city.ico</normaloff>:/icon/city.ico</iconset>
          </attribute>
          <attribute name="title">
           <string>Город 3</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_4">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Текущая цена:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="current_price_lineEdit_3">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Себестоимость:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="first_cost_lineEdit_3"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Минимальная цена:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLineEdit" name="min_price_lineEdit_3">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>%</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="min_price_spinBox_3">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
               <property name="value">
                <double>25.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Максимальная цена:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLineEdit" name="max_price_lineEdit_3">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>%</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="max_price_spinBox_3">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
               <property name="value">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="sity_4">
          <attribute name="icon">
           <iconset resource="../media/add_data_to_base_rs.qrc">
            <normaloff>:/icon/city.ico</normaloff>:/icon/city.ico</iconset>
          </attribute>
          <attribute name="title">
           <string>Город 4</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_5">
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Текущая цена:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="current_price_lineEdit_4">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>Себестоимость:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="first_cost_lineEdit_4"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>Минимальная цена:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLineEdit" name="min_price_lineEdit_4">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>%</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="min_price_spinBox_4">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
               <property name="value">
                <double>25.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Максимальная цена:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLineEdit" name="max_price_lineEdit_4">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>%</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="max_price_spinBox_4">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
               <property name="value">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../media/add_data_to_base_rs.qrc"/>
 </resources>
 <connections/>
</ui>
